.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  root: true,
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module'
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:ember-suave/recommended'
```

Comment:
Yeah, that's a good point. We should probably switch to the new shims anyway which make most of the destructuring unnecessary anyway. For now I think following a consistent style across all of our projects is benefitial though.